Base.rte/AI/HumanFunctions.lua

```diff
@@ -160,15 +160,15 @@ function HumanFunctions.DoVisibleInventory(actor, showAll)
 						local drawPos = actor.Pos + Vector((-actorBack.X * 0.5 - stackX) * actor.FlipFactor, -actorBack.Y * 0.75):RadRotate(actor.RotAngle);
 						--Display tall objects upright
 						local widthToHeightRatio = ToMOSprite(item):GetSpriteWidth()/ToMOSprite(item):GetSpriteHeight();
-						local tallAng = widthToHeightRatio > 1 and 1.57 or 0;
+						local orientation = widthToHeightRatio > 1 and 1.57 * actor.FlipFactor or 0;
```

Comment:
Won't this rotate it to be upside down when the actor is flipped? Or am I confused and it just wasn't working properly before?

Reply:
The items are tilted 90° to begin so each flipping means the other one when it's upright. Doesn't change how it looks compared to before other than the bitmap is consistently mirrored along with the actor.